### DISCUSSION TITLE
Resource policy not sending paramenter

### DISCUSSION THREAD

**Discussion by samuelhgf** [laravel/framework#32844] [General]
- Laravel Version: 7.1
- PHP Version: 7.4

### Description:

Hi, I am using a resource policy for my controllers.

In each controller __construct I call the method authorizeResource passing the model and the parameter name that should came from request

`public function __construct()
    {
        $this->authorizeResource(Phone::class, 'phone');
    }`

When it call a method that need the parameter from request it throws an error:

<img width="542" alt="Captura de Tela 2020-05-16 às 11 51 02" src="https://user-images.githubusercontent.com/16768378/82122705-8791de00-976b-11ea-9bc1-af6134684513.png">

### Steps To Reproduce:

Create a controller and a policy

In the controller construct call the method authorizeResource passing the model and the parameter name that came from request

**Comment by mo3000** [comment]
Have you tried this ? [Supplying Additional Context](https://laravel.com/docs/7.x/authorization#supplying-additional-context)
